src/modules/compliance/src/lib/Regex.h

```diff
@@ -7,62 +7,23 @@
 // It has to be removed and forgotten after we abandon support
 // for older GCC versions.
 
-#if __cplusplus >= 201103L &&                                                                                                                          \
-    (!defined(__GLIBCXX__) || (__cplusplus >= 201402L) ||                                                                                              \
-        (defined(_GLIBCXX_REGEX_DFS_QUANTIFIERS_LIMIT) || defined(_GLIBCXX_REGEX_STATE_LIMIT) || (defined(_GLIBCXX_RELEASE) && _GLIBCXX_RELEASE > 4)))
-
-#include <regex>
-#define regex_search std::regex_search
-using regex = std::regex;
+#ifndef COMPLIANCE_REGEX_H
+#define COMPLIANCE_REGEX_H
 
+#if __cplusplus >= 201103L && ((__cplusplus >= 201402L) || (defined(_GLIBCXX_REGEX_DFS_QUANTIFIERS_LIMIT) || defined(_GLIBCXX_REGEX_STATE_LIMIT) ||    \
+                                                               (defined(_GLIBCXX_RELEASE) && _GLIBCXX_RELEASE > 4)))
```

Comment:
nit: this line looks a bit strange, may want  either yo not break or to align broken part 4 spaces to left